src/components/content/resource/helpers/details-card.tsx

```diff
@@ -73,6 +75,11 @@ export const recursivelyTransposeTables = (node: ReactNode, curDepth: number): R
 };
 
 const transposeTable = (tbl: ReactElement): ReactElement => {
+  // if there's no children, then return with original tbl
+  if (!tbl.props || !tbl.props.children || !Array.isArray(tbl.props.children)) {
+    return tbl;
+  }
```

Comment:
Can probably simplify to
```suggestion
  if (!Array.isArray(tbl.props?.children)) {
    return tbl;
  }
```